operations/services.md

```diff
@@ -1579,6 +1579,311 @@ Returns all companionships based on customers, reservations or reservation group
+| `Floor` | [Bool update value](reservations.md#bool-update-value) | optional | Specify whether [Resource access token](#resource-access-token) grants permission to access floor. |
+| `Building` | [Bool update value](reservations.md#bool-update-value) | optional | Specify whether [Resource access token](#resource-access-token) grants permission to access building. |
+
+### Response
+
+```javascript
+{
+    "ResourceAccessTokens": [
+        {
+            "Id": "72d4b117-1f84-44a3-1f84-8b2c0635ac60",
+            "ServiceOrderId": "35d4b117-4e60-44a3-9580-c582117eff98",
+            "CompanionshipId": "25d4b117-4e60-44a3-9580-c582117eff98",
+            "ResourceId": "65d4b117-4e60-44a3-9580-c582117eff98",
+            "Type": "PinCode",
+            "Value": "1234#",
+            "SerialNumber": null,
+            "ValidityStartUtc": "2020-10-09T22:00:00Z",
+            "ValidityEndUtc": "2020-10-10T22:00:00Z",
+            "Permissions":
+            {
+                "Bed": true,
+                "Room": false,
+                "Floor": false,
+                "Building": false
+            }
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `ResourceAccessTokens` | array of [Resource access token](#resource-access-token) | required | Resource access tokens. |
+
+## Update resource access tokens
+
+Updates the [Resource access token](#resource-access-token) validity interval and [permissions](#resource-access-token-permission-parameter) that its grants.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/resourceAccessTokens/update`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "ResourceAccessTokenUpdates": [
+        {
+            "ResourceAccessTokenId": "35d4b117-4e60-44a3-9580-c582117eff98",
+            "ValidityStartUtc": { "Value": "2020-10-09T22:00:00Z" },
+            "ValidityEndUtc": { "Value": "2020-10-10T22:00:00Z" },
+            "Permissions":
+            {
+                "Bed": { "Value": true },
+                "Room": { "Value": false },
+                "Floor": { "Value": false },
+                "Building": { "Value": false }
+            }
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `ResourceAccessTokenUpdates` | Array of [Resource access token update](#resource-access-token-update) | required | Parameters of [Resource access token](#resource-access-token). |
```

Comment:
Not sure if this should be described same as create paramters. Please check similar methods and use consistent naming conventions if needed.